.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules/
 yarn-error.log
 yarn.lock
+.vscode/
```

Comment:
I can see you added .vscode in gitignore but it is still in your repo. Please resolve this